sdk/search/search-documents/samples/javascript/src/dataSourceConnections/createDataSourceConnection.js

```diff
@@ -27,4 +32,6 @@ async function main() {
   await client.createDataSourceConnection(dataSourceConnection);
 }
 
-main();
+main().catch((err) => {
```

Comment:
I think catching any errors and printing them is good, since Node's "unhandled promise rejection" feature is a bit messy right now